tracing/jaeger/src/main/java/io/helidon/tracing/jaeger/JaegerTracerBuilder.java

```diff
@@ -640,13 +640,30 @@ private void flushIntervalMs(Long aLong) {
         this.reporterFlushIntervalMillis = aLong;
     }
 
-    enum SamplerType {
+    /**
+     * Sampler type definition.
+     * Available options are "const", "probabilistic", "ratelimiting" and "remote".
+     */
+    public enum SamplerType {
+        /**
+         * Constant option.
+         */
         CONSTANT("const"),
+        /**
+         * Probabilistic option.
+         */
         PROBABILISTIC("probabilistic"),
+        /**
+         * Ratelimiting option.
+         */
         RATE_LIMITING("ratelimiting"),
+        /**
+         * Remote option.
+         */
         REMOTE("remote");
```

Comment:
If the constant is used in the builder, javadoc should describe what are given options actually doing, which one is the default and to which options sampler type relates(samplerParam) see Jaeger doc https://www.jaegertracing.io/docs/1.29/sampling/

Reply:
Fixed :)